src/main.cpp

```diff
@@ -843,6 +844,8 @@ unsigned int GetP2SHSigOpCount(const CTransaction& tx, const CCoinsViewCache& in
 bool CheckTransaction(const CTransaction& tx, CValidationState &state,
                       libzcash::ProofVerifier& verifier)
 {
+    // FIXME: Check transaction with context - check against contect of local accepted branch here?
```

Comment:
The target branch ID is probably sufficient context; see my later comment.

Reply:
I don't understand this comment.